controllers/configuration_controller.go

```diff
@@ -749,7 +750,7 @@ func (meta *TFConfigurationMeta) assembleTerraformJob(executionType TerraformExe
 		meta.ResourcesRequestsCPU != "" || meta.ResourcesRequestsMemory != "" {
 		resourceRequirements := v1.ResourceRequirements{}
 		if meta.ResourcesLimitsCPU != "" || meta.ResourcesLimitsMemory != "" {
-			resourceRequirements.Limits = v1.ResourceList(map[v1.ResourceName]resource.Quantity{})
+			resourceRequirements.Limits = map[v1.ResourceName]resource.Quantity{}
```

Comment:
Any problems here?

Reply:
`v1.ResourceList` is a redundant type conversion, so I removed it.